ska/__main__.py

```diff
@@ -1,55 +1,96 @@
 """ska: split k-mer alignment
 
 Usage:
-  ska fasta <files>... -o <output>
-  ska align -l <file-list> -o <output>
-  ska map ref -l <file-list> -o <output>
-  ska (-h | --help)
-  ska (--version)
+  __main__.py --fasta <fasta>
+  __main__.py --align <align>
+  __main__.py --map <map>
 
 Options:
-  -h --help     Show this help.
-  --version     Show version.
+  -h --help         Show this help.
+  --version         Show version.
+  -kmer_size=<int>  Size of k-mers; needs to be off (default: 7)
 
-  -o <output>    Output prefix.
-  -l <file-list> File with a list of input files.
 """
+# -d --directory    Directory of fasta files (files have to end in *.fa or *.fasta)
```

Comment:
If you want to leave this in, leave a comment explaining why (e.g. is it the eventual CLI you want to aim for?)